cleanlab/experimental/label_issues_batched.py

```diff
@@ -0,0 +1,465 @@
+        ``METHOD1="low_normalized_margin"``, ``METHOD2="normalized_margin"``.
+
+        Note: The estimated number of issues may differ from :py:func:`filter.find_label_issues <cleanlab.filter.find_label_issues>`
+        by 1 due to rounding differences.
+
+        Returns
+        -------
+        issue_indices : np.ndarray
+          Indices of examples with label issues, sorted by label quality score.
+        """
+        if not self.store_results:
+            raise ValueError(
+                "Must initialize the LabelInspector with `store_results` == True. "
+                "Otherwise you can identify label issues yourself based on the scores from all "
+                "the batches of data and the total number of issues returned by `get_num_issues()`"
+            )
+        if self.examples_processed_quality < 1:
+            raise ValueError(
+                "Have not evaluated any labels yet. Call `score_label_quality()` first."
+            )
+        if self.verbose:
+            print(
+                f"Total number of examples whose labels have been evaluated: {self.examples_processed_quality}"
+            )
+        return find_top_issues(self.get_quality_scores(), top=self.get_num_issues())
+
+    def update_confident_thresholds(self, labels: LabelLike, pred_probs: np.ndarray):
+        """
+        Updates the estimate of confident_thresholds stored in this class using a new batch of data.
+        Inputs should be in same format as for: :py:func:`count.get_confident_thresholds <cleanlab.count.get_confident_thresholds>`.
+
+        Parameters
+        ----------
+        labels: np.ndarray or list
+          Given class labels for each example in the batch, values in ``0,1,2,...,K-1``.
+
+        pred_probs: np.ndarray
+          2D array of model-predicted class probabilities for each example in the batch.
+        """
+        labels = _batch_check(labels, pred_probs, self.num_class)
+        batch_size = len(labels)
+        batch_thresholds = get_confident_thresholds(
+            labels, pred_probs
+        )  # values for missing classes may exceed 1 but should not matter since we multiply by this class counts in the batch
+        batch_class_counts = value_counts_fill_missing_classes(labels, num_classes=self.num_class)
+        self.confident_thresholds = (
+            self.examples_per_class * self.confident_thresholds
+            + batch_class_counts * batch_thresholds
+        ) / np.clip(
+            self.examples_per_class + batch_class_counts, a_min=1, a_max=None
+        )  # avoid division by 0
+        self.examples_per_class += batch_class_counts
+        self.examples_processed_thresh += batch_size
+
+    def score_label_quality(self, labels: LabelLike, pred_probs: np.ndarray) -> np.ndarray:
+        """
+        Updates the estimate of confident_thresholds stored in this class using a new batch of data.
```

Comment:
Docstring meant for `update_confident_thresholds()`

Reply:
good catch, fixed it